### ISSUE TITLE
Regexp parser error

### ISSUE THREAD

**Issue by zim32** [GregRos/parjs#36]
Trying all this leads to error

const { regexp } = require('parjs');

let TEST = regexp(new RegExp('a'));
let TEST = regexp(/a/);
let TEST = regexp('a');
TEST.parse('a');

```
this.reason = `expecting input matching /${origRegexp.source}/`;
                ^
TypeError: Cannot set property 'reason' of undefined
    at regexp (/home/***/www/node_modules/parjs/internal/parsers/regexp.js:19:17)
    at Object.<anonymous> (/home/***/www/parser.js:14:14)
    at Module._compile (internal/modules/cjs/loader.js:1138:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1158:10)
    at Module.load (internal/modules/cjs/loader.js:986:32)
    at Function.Module._load (internal/modules/cjs/loader.js:879:14)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:71:12)
    at internal/main/run_main_module.js:17:47

```

**Comment by zim32** [comment]
This works. Something with typescript export import

const p = require('parjs');
let TEST = p.regexp(/a/);

**Comment by rapropos** [comment]
I believe this is addressed by [PR 30](https://github.com/GregRos/parjs/pull/30). Here's how I worked around it:

```
import {regexp as buggyRegexp} from "parjs";
let regexp: (origRegexp: RegExp) => Parjser<string[]> = buggyRegexp.bind({});
```

**Comment by GregRos** [comment]
Fixed in #30. Thank you!
